src/backend/api/reviews.js

```diff
@@ -0,0 +1,80 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+
+// GET all reviews
+router.get("/", async (req, res) => {
+  try {
+    const reviews = await knex("reviews").select("*");
+    res.json(reviews);
+  } catch (error) {
+    console.error(error);
+    res.status(500).json({ error: "Internal server error" });
+  }
+});
+
+// GET all reviews for a specific meal
+router.get("/meals/:meal_id", async (req, res) => {
+  const mealId = req.params.meal_id;
+  try {
+    const reviews = await knex("reviews").where({ meal_id: mealId });
+    res.json(reviews);
+  } catch (error) {
+    console.error(error);
+    res.status(500).json({ error: "Internal server error" });
+  }
+});
```

Comment:
This method should be in the `meals.js` file, since the URL starts with `api/meals`. Thus, once you move it to the `meals.js` file, there the path will be: `mealsRouter.get("/:meal_id/reviews", ...)`